cypress/data/index.js

```diff
@@ -22,23 +22,14 @@ export const TEST_AOS = [
     },
 ]
 
-// alpha-numerics
+// alphanumerics
 export const TEST_DIM_TEXT = 'Analytics - Text'
```

Comment:
What do you think about dropping the `TEST_` prefix? They are the real dimensions. Instead they could be called `DIMENSION_TEXT` or `DIM_TEXT`. Technically these (until line 59) are data elements, though maybe we don't want to get that specific.